testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-secrets-provider/secrets.toml

```diff
@@ -0,0 +1,5 @@
+[keystore]
+secrets-provider.name = "keystore"
+keystore-file-path = "/docker/presto-product-tests/conf/presto/etc/credential.jckes"
```

Comment:
you can add some REAMDE file next to `credential.jckes` so one would know how to recreate it, just in case